obabeliface/obabeliface.cpp

```diff
@@ -190,6 +190,9 @@ namespace Molsketch
 
   // TODO should be const, but OpenBabel iterator methods do not support const
   bool hasCoordinates(OpenBabel::OBMol &molecule) {
+#if (OB_VERSION >= OB_VERSION_CHECK(3, 0, 0))
```

Comment:
```suggestion
#if (OB_VERSION >= OB_VERSION_CHECK(3, 1, 1))
```
The current version is 3.1.1, no?

Reply:
Yes, but it compiled with 3.0.0 too (tested). I thought keep the existing condition would be ok, as it's not really 3.1.1 either.

Reply:
Hm... interesting. Ok, thank you for checking and sorry for the wait. I'll check with the next release built then and merge for now.